sites/site-utilities/src/definitions/fast-components/fast-tooltip.definition.ts

```diff
@@ -1,62 +0,0 @@
-import { WebComponentDefinition } from "@microsoft/fast-tooling/dist/esm/data-utilities/web-component";
-import { DataType } from "@microsoft/fast-tooling";
-
```

Comment:
This isn't needed?

Reply:
No, no other component has one here.   I think it was a holdover from tooltip being added to the depot while the definition file location was changed.